.azure/scripts/documentation.yml

```diff
@@ -0,0 +1,24 @@
+# This task tries to verify if the documentation will build properly on RTD
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.8'
+
+- script: |
+    pip install  --exists-action=w --no-cache-dir -r test-requirements.txt -r doc/doc-requirements.txt
+  displayName: 'Install requirements'
+
+- script: |
+    python -m sphinx -T -b readthedocs -W -d _build/doctrees-readthedocs -D language=en doc build/html
```

Comment:
I have added -W in the past to turn warnings into errors, as most of time these are mistakes in the markup. Now I propose to add the flag --keep-going as it seems to collect all warnings first and then exit with status -1.

from the sphinx documentation:
```

-W

    Turn warnings into errors. This means that the build stops at the first warning and sphinx-build exits with exit status 1.

--keep-going

    With -W option, keep going processing when getting warnings to the end of build, and sphinx-build exits with exit status 1.

    New in version 1.8.
```

Reply:
Okay that is on a CI task, so we can do that in master and I will let the release go without updating as this file is not in the release itself.